src/tikzdocument.jl

```diff
@@ -105,6 +105,17 @@ end
 
 _OLD_LUALATEX = false
 
+"""
+List of class options used in the preamble (default ["tikz"]).
+
+By setting
+`PGFPlotsX.CLASS_OPTIONS[1] = "varwidth"; push!(PGFPlotsX.CLASS_OPTIONS, "crop=false")`
+the preamble will contain `documentclass[varwidth,crop=false]{standalone}`.
+
+See https://www.ctan.org/pkg/standalone for a list of options.
+"""
+CLASS_OPTIONS = ["tikz"]
```

Comment:
I usually make globals `const`, but here it would not matter much (up to you).

Reply:
I thought I would do it in the same style as the `CUSTOM_PREAMBLE` etc. in `build.jl`; these are all non-constants. But I am happy to change this, if you prefer.

Reply:
Don't. I think the PR is fine as is.